src/service/light.rs

```diff
@@ -495,6 +495,10 @@ pub enum EffectType {
     Sparkle,
     Fire,
     Candle,
+    Underwater,
+    Cosmos,
+    Sunbeam,
+    Enchant,
     NoEffect,
```

Comment:
Need to start doing this in more places to remain robust against the Hue team's total disregard for semantic versioning 😆 

```suggestion
    NoEffect,
    #[serde(other)]
    Unknown,
```